tests/cucumber/features/step_definitions/organizations_steps.coffee

```diff
@@ -40,7 +40,7 @@ do ->
 
     @Then /^I should be on the "([^"]*)" detail page$/, (name, callback) ->
       @browser
-        .waitForVisible('.organization-detail', assert.ifError)
+        .waitForVisible('.organization-detail', 30000, assert.ifError)
```

Comment:
It could be annoying if we're running the test locally and something's actually broken - we'd have to wait 30 seconds for it to fail.